config/bollgrid.yaml

```diff
@@ -52,11 +52,12 @@ backtest:
   symbols:
   - BTCUSDT
   account:
-    makerCommission: 15
-    takerCommission: 15
-    balances:
-      BTC: 0.0
-      USDT: 10000.0
+    max:
+      makerCommission: 15
+      takerCommission: 15
```

Comment:
makerCommission and takerCommission is not used anymore, we now use the makerFeeRate and takerFeeRate field in the exchange session struct.